packages/web-react/src/utils/strings.ts

```diff
@@ -0,0 +1,2 @@
+export const toPascalCase = (string: string) =>
+  string.replace(/\w+/g, (word) => word[0].toUpperCase() + word.slice(1).toLowerCase()).replace('-', '');
```

Comment:
Maybe `(word) => ${word[0].toUpperCase()}${word.slice(1).toLowerCase()).replace('-', '')}`?

Reply:
This file was removed.